build/assets/scripts/runc-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# "0" means "disabled"
+if [ "{{.CPUMask}}" == "0" ]; then
+	exec /usr/bin/runc "$@"
+fi
+if [ -n "$3" ] && [ "$3" == "start" ]; then
+        if [ -n "$4" ]; then
+                bundle=$(/bin/runc state "$4" 2>/dev/null | jq .bundle -r)
+                if [ -n "$bundle" -a -f "$bundle/config.json" ]; then
+                        command=$(grep "/usr/bin/pod" "$bundle/config.json")
+                        if [ -n "$command" ]; then
+                                cpuset=$(find /sys/fs/cgroup/cpuset/ -name cpuset.cpus | grep $4)
+                                echo "{{.CPUMask}}" > $cpuset
+                        fi
+                fi
+        fi
+fi
+taskset -c "{{.CPUMask}}" /usr/bin/runc "$@"
```

Comment:
do we want to use `exec` here?

Reply:
because it makes the wrapper disappear once its work is done. This can impact (unix) signal delivery and (unix)session management, but in practical terms it should make no difference in our case because `runc` is not supposed to stay running for long time. We will need to spawn a shell anyway and we have the some issues (or lack thereof) with environment passing. So it's probably not worth considering the looming code freeze.

For illustrative purposes, let's consider:
```
$ cat sleep.sh 
#!/bin/sh
sleep 10h
```

this gives us this process tree:
```
fromani     4480  0.3  0.4 701036 70692 ?        Ssl  08:32   2:18  \_ /usr/libexec/gnome-terminal-server
fromani     4487  0.0  0.0 220432  7144 pts/0    Ss+  08:32   0:01  |   \_ bash
fromani    17017  0.0  0.0 220400  7252 pts/1    Ss   10:32   0:02  |   \_ bash
fromani    70704  0.0  0.0 216732  3016 pts/1    S+   18:35   0:00  |   |   \_ /bin/sh ./sleep.sh
fromani    70705  0.0  0.0 215296   520 pts/1    S+   18:35   0:00  |   |       \_ sleep 10h
```
(note the intermediate `/bin/sh`)

now with exec:
```
#!/bin/sh
exec /bin/sleep 10h
```

this gives us:

```
fromani     4480  0.3  0.4 701036 70752 ?        Ssl  08:32   2:19  \_ /usr/libexec/gnome-terminal-server
fromani     4487  0.0  0.0 220432  7144 pts/0    Ss+  08:32   0:01  |   \_ bash
fromani    17017  0.0  0.0 220400  7252 pts/1    Ss   10:32   0:02  |   \_ bash
fromani    70791  0.0  0.0 215296   580 pts/1    S+   18:35   0:00  |   |   \_ /bin/sleep 10h
```
(note the intermediate `/bin/sh` is gone)